cmd/podman/containers/ps.go

```diff
@@ -180,24 +176,22 @@ func ps(cmd *cobra.Command, args []string) error {
 	switch {
 	case report.IsJSON(listOpts.Format):
 		return jsonOut(listContainers)
-	case listOpts.Quiet:
+	case listOpts.Quiet && !cmd.Flags().Changed("format"):
```

Comment:
Should we just throw an error in this situation.  Then you wouldn't need to check later.

If the user specifies --quiet and --format, we should tell them  this is not allowed.

Reply:
I think that we have to allow this because Docker did, and people actually came to expect that it worked...

Generally speaking, `--format` overrides `--quiet`.